open-authoring/open-authoring/OpenAuthoringProvider.tsx

```diff
@@ -80,13 +95,6 @@ export const OpenAuthoringProvider = ({
     }
   }, [authorizing, forkValid, authenticated])
 
-  // Hook to update root openAuthoring state when form fails.
-  // We need to perform to check before an action is clicked (e.g start auth flow)
-  // Because if it is perform on-the-fly, the window may be blocked.
-  useEffect(() => {
```

Comment:
I think we'll still want this, it's for when we show an error-modal we will want to get the latest authentication state